### PR TITLE
Hover window

### PR DESCRIPTION
**Pull request by miahsanchez**
When you scroll over a marker an infoWindow pops up (still with the hardcoded data). And disappears when you scroll out.

### CODE REVIEW

File: project/frontend/src/components/ResultsMap.js [hunk 1]
```diff
@@ -1,12 +1,55 @@
...
+      position: 'relative',
+      bottom: 80,
+      left: '-45px',
+      width: 220,
+      backgroundColor: 'white',
+      boxShadow: '0 2px 7px 1px rgba(0, 0, 0, 0.3)',
+      padding: 10,
+      fontSize: 14,
+      zIndex: 100,
+    };
```
Comment (miahsanchez) [inline project/frontend/src/components/ResultsMap.js:42]: for some reason i put that im comparing this branch to my addInfoWindows branch which has this code but it still says that this is new to this branch ? confusion

Comment (jesseelizabeth) [reply project/frontend/src/components/ResultsMap.js:42]: Did you branch from the addInfoWindows branch to create this branch?

Comment (miahsanchez) [reply project/frontend/src/components/ResultsMap.js:42]: I did!

File: project/frontend/src/components/ResultsMap.js [hunk 2]
```diff
@@ -1,12 +1,54 @@
...
 
 function MapContainer(props) {
   const restaurants = props.restaurants;
   const userCenter = props.userLocation;
+  const [activeMarker, setActiveMarker] = useState({});
+  const [showInfoWindows, setShowInfoWindows] = useState([
+    false,
+    false,
+    false,
+    false,
```
Comment (surbhimadan1995) [inline project/frontend/src/components/ResultsMap.js:13]: for this, I think we should add the param name for the state to make it clear what the state is referring to 
name1: false,
name2: false 

etc

**Pull request closed without merging**